x-pack/plugins/security_solution/public/detections/hooks/trigger_actions_alert_table/use_cell_actions.tsx

```diff
@@ -62,20 +62,17 @@ export const getUseCellActionsHook = (tableId: TableId) => {
       tableDefaults.viewMode;
 
     const cellActionProps = useMemo<UseDataGridColumnsSecurityCellActionsProps>(() => {
-      const fields =
-        viewMode === VIEW_SELECTION.eventRenderedView
-          ? []
-          : columns.map((col) => {
-              const fieldMeta: Partial<BrowserField> | undefined = browserFieldsByName[col.id];
-              return {
-                name: col.id,
-                type: fieldMeta?.type ?? 'keyword',
-                values: (finalData as TimelineNonEcsData[][]).map(
-                  (row) => row.find((rowData) => rowData.field === col.id)?.value ?? []
-                ),
-                aggregatable: fieldMeta?.aggregatable ?? false,
-              };
-            });
+      const fields = columns.map((col) => {
```

Comment:
Do we need to calculate fields and make `useDataGridColumnsSecurityCellActions` process all actions, when `viewMode === VIEW_SELECTION.eventRenderedView`? These actions are discarded later at L92.
Was the current implementation causing the problem?

Reply:
@semd Do you have any issue, If I proceed with this workaround and I will follow up about the above bug separately.

Reply:
Okay. I thought we had fixed this problem the last time we talked about it. 
I am okay with the workaround you implemented. But if changing the way we call this hook is solving the error, it means that there's some problem in the cell-actions.
Would you be willing to investigate it separatelly?